src/components/Forms/Input/input.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useField } from 'formik';
+
+export const Input = ({
+  id, label, ...props
+}) => {
+  const [field] = useField(props);
+
+  return (
+    <div className="form__field">
+      <label className="form__label" htmlFor={id}>{label}</label>
+      <input className="form__input" {...field} {...props} />
```

Comment:
also it will be better to add `id` property here, cause you distructure it from `props`

Reply:
fixed, thanks